compiler-rt/lib/tsan/rtl/tsan_rtl_report.cpp

```diff
@@ -199,6 +198,55 @@ void ScopedReportBase::AddMemoryAccess(uptr addr, uptr external_tag, Shadow s,
   }
 }
 
+void ScopedReportBase::SymbolizeStackElems() {
+  // symbolize memory ops
+  for (usize i = 0; i < rep_->mops.Size(); i++) {
```

Comment:
Nits throughout this function: https://llvm.org/docs/CodingStandards.html#don-t-evaluate-end-every-time-through-a-loop